app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -130,6 +128,11 @@ class SignUpFragment : Fragment() {
         Snackbar.make(rootView, R.string.logged_in_automatically, Snackbar.LENGTH_SHORT).show()
     }
 
+    private fun Dialog.show(show: Boolean) {
+        if (show && !this.isShowing) this.show()
+        else if (!show && this.isShowing) this.dismiss()
+    }
```

Comment:
Why are there 2 methods in 2 activities doing the same thing?

Reply:
Resolved